src/awesomeSezzle.js

```diff
@@ -622,7 +622,7 @@ class AwesomeSezzle {
     const price = this.parseMode ==='default' ? HelperClass.parsePrice(priceText) : HelperClass.parsePrice(priceText,this.parseMode);
 	const formatter =  priceText.replace(priceString, '{price}');
 	const terms = this.termsToShow(price);
-	const sezzleInstallmentPrice = this.isProductEligibleLT(amount) ? this.calculateInterest(price,terms[terms.length - 1],this.bestAPR).toFixed(2): (price / this.numberOfPayments).toFixed(2);
+	const sezzleInstallmentPrice = this.isProductEligibleLT(amount) ? this.calculateMonthlyWithInterest(price,terms[terms.length - 1],this.bestAPR) : (price / this.numberOfPayments).toFixed(2);
```

Comment:
We probably don't need `toFixed(2)` in `(price / this.numberOfPayments).toFixed(2)` too